apps/src/aichat/views/modelCustomization/CompareModelsDialog.tsx

```diff
@@ -38,17 +39,20 @@ const CompareModelsDialog: React.FunctionComponent<{
         <ModelDescriptionPanel
           initialSelectedModelId={chosenModelLeft}
           availableModels={availableModels}
-          dropdownName="choose-model-1"
+          dropdownName={aichatI18n.modelComparisonSlot1()}
```

Comment:
double check this - looks like a key or something that we don't want to translate